app/src/main/kotlin/com/exactpro/th2/lwdataprovider/grpc/GrpcDataProviderImpl.kt

```diff
@@ -244,7 +244,7 @@ open class GrpcDataProviderImpl(
         }
     }
 
-    protected open fun <T> processResponse(
+    open fun <T> processResponse(
```

Comment:
suggestion: Let's mark it as `internal`. It does not change much but at lease it is a marker that we don't want to expose this